src/styles/myStyles.css

```diff
@@ -0,0 +1,7 @@
+.centered {
```

Comment:
If we have .less now we should convert all css files to .less files since less compiles into css with extra features